lib/src/interop/storage_interop.dart

```diff
@@ -76,7 +76,6 @@ class FullMetadataJsImpl extends UploadMetadataJsImpl {
 @anonymous
 class UploadMetadataJsImpl extends SettableMetadataJsImpl {
   external String get md5Hash;
-  external void set md5Hash(String s);
```

Comment:
Ah...k.

I saw no uses of it. Figured folks would set it in a constructor when using it.

I can add it back, though...to keep consistency.